### DISCUSSION TITLE
Creating Detached Versions?

### DISCUSSION THREAD

**Discussion by altezza04** [directus/directus#3770] [Archive]
What's detached versions? Dose it the same as  'Save a Copy' option in the item editor? 

https://docs.directus.io/concepts/activity-and-versions.html#activity
Creating Detached Versions
You can also create a new version for an item without saving the data to the parent item itself. This allows you to "stage" changes to an item that may already be live/published.

#

**Comment by benhaynes** [comment]
This is actually an _upcoming_ feature, and should probably be commented out of the docs until it's in. Basically it's a way to create a "revision" in the future instead of the past... so it won't change the live/active item in the DB (yet).

**Reply by rijkvanzanten** [reply]
@philleepflorence

**Reply by enzolupia** [reply]
@benhaynes  @rijkvanzanten Hy guys, is this feature already available or will it be in the near future?

**Reply by rijkvanzanten** [reply]
> @benhaynes @rijkvanzanten Hy guys, is this feature already available or will it be in the near future?

It's an open feature request 👉🏻 https://github.com/directus/directus/discussions/2975
